src/components/widgets/accounts-cash-projection/accounts-cash-projection.directive.coffee

```diff
@@ -103,6 +103,37 @@ module.controller('WidgetAccountsCashProjectionCtrl', ($scope, $q, $filter, $tim
       trxId
     ).then(-> $scope.trxList.updated = true)
 
+  $scope.trxList.includeSchedulableTransactions = (resourcesType, trx) ->
+    BoltResources.update(
+      w.metadata.bolt_path,
+      resourcesType,
+      trx.id,
+      {
+        recurring: trx.recurring,
+        recurring_pattern: trx.recurring_pattern,
+        recurring_end_date: if trx.recurring_end_date then moment(trx.recurring_end_date).format('YYYY-MM-DD') else null
+      }
+    ).then(->
+      $scope.trxList.updated = true
+      $scope.trxList.fetch()
+    )
+
+  $scope.trxList.deleteSchedulableTransactions = (resourcesType, trx) ->
+    trxId = trx.recurring_parent || trx.id
+    trx = _.find($scope.trxList.transactions, (trx) -> trx.id == trxId)
+    _.remove($scope.trxList.transactions, (trx) -> trx.recurring_parent == trxId)
```

Comment:
Where is `recurring_parent` set on the `trx`? I cant find it using search. What is `recurring_parent`? Why is `_.remove` called twice? 
Maybe it could be a bit clearer with at least comments on why `if trx.status == 'FORECAST '` `_.remove` will be called twice.

Reply:
I included a comment to be clearer... it's no easy to follow the idea... I spend few seconds to remember too. For sure it'll be easy for another developer understand now.